homeassistant/components/airzone/__init__.py

```diff
@@ -26,17 +26,20 @@ def __init__(
         coordinator: AirzoneUpdateCoordinator,
         entry: ConfigEntry,
         system_zone_id: str,
-        zone_name: str,
+        zone_data: dict,
     ) -> None:
         """Initialize."""
         super().__init__(coordinator)
 
         self._attr_device_info: DeviceInfo = {
             "identifiers": {(DOMAIN, f"{entry.entry_id}_{system_zone_id}")},
             "manufacturer": MANUFACTURER,
-            "name": f"Airzone [{system_zone_id}] {zone_name}",
+            "name": f"Airzone [{system_zone_id}] {zone_data[AZD_NAME]}",
         }
+        self.airzone: AirzoneLocalApi = coordinator.airzone
```

Comment:
We don't need to store this? The coordinator is already available everywhere, as this class extends the CoordinatorEntity

Reply:
Removed in 66bd22912bbcf6577569050a66093a5dd4394fab